core/Archive/ArchiveInvalidator.php

```diff
@@ -90,7 +103,21 @@ public function rememberToInvalidateArchivedReportsLater($idSite, Date $date)
             // that happen to run in overlapping transactions.
             $mykey = $this->buildRememberArchivedReportIdProcessSafe($idSite, $date->toString());
             Option::set($mykey, '1');
+            Cache::clearCacheGeneral();
+        }
+    }
+
+    private function getRememberedArchivedReportsOptionFromTracker($idSite, $dateStr)
+    {
+        $cacheKey = self::TRACKER_CACHE_KEY;
```

Comment:
Ideally, it would check whether we are in tracker mode and then read it from cache, or otherwise use the result directly. Otherwise we have to basically launch Matomo in tracker mode during a regular request. Seems to be eg used in DataSubjects which is not a tracking request.